Editor/Resources/EditorWindow/Pages/AnywherePage.uxml

```diff
@@ -0,0 +1,102 @@
+                    <ui:TextField name="AnywherePageCreateFleetInput" class="text-input" value="GameName-AnywhereFleet"/>
+                    <ui:Label name="AnywherePageCreateFleetHint" class="input-hint" text="Fleet Name must have 1–1024 characters. Valid characters are A-Z, a-z, 0-9, _ and - (hyphen)"/>
+                </ui:VisualElement>
+            </ui:VisualElement>
+            <ui:VisualElement class="form-row">
+                <ui:VisualElement class="form-row__input form-row__input--no-label">
+                    <ui:Button name="AnywherePageCreateFleetButton" class="button button--primary" text="Create Anywhere Fleet"/>
+                    <ui:Button name="AnywherePageCreateFleetCancelButton" class="button" text="Cancel"/>
+                </ui:VisualElement>
+            </ui:VisualElement>
+        </ui:Foldout>
+        <ui:Foldout name="AnywherePageConnectFleetTitle" class="foldout" text="Connect to Anywhere Fleet">
+            <ui:VisualElement name="AnywhereFleetName" class="form-row">
+                <ui:Label name="AnywherePageConnectFleetLabel" class="form-row__label" text="Fleet Name"/>
+                <ui:DropdownField name="AnywherePageConnectFleetDropdown" class="form-row__input dropdown"/>
+            </ui:VisualElement>
+            <ui:VisualElement name="AnywherePageConnectFleetID" class="form-row form-row__hidden">
+                <ui:Label name="AnywherePageConnectFleetIDLabel" class="form-row__label" text="Fleet ID"/>
+                <ui:Label name="AnywherePageConnectFleetIDDisplay" class="form-row__text-display" text="1234567890"/>
+            </ui:VisualElement>
+            <ui:VisualElement name="AnywherePageConnectFleetStatus" class="form-row form-row__hidden">
+                <ui:Label name="AnywherePageConnectFleetStatusLabel" class="form-row__label" text="Fleet status"/>
+                <ui:Label name="AnywherePageConnectFleetStatusDisplay" class="form-row__text-display" text="Connected"/>
+            </ui:VisualElement>
+            <ui:VisualElement name="AnywhereFleetButtons" class="form-row form-row__hidden">
+                <ui:VisualElement class="form-row__input form-row__input--no-label">
+                    <ui:Button name="AnywherePageConnectFleetNewButton" class="button button--primary" text="Create New Anywhere Fleet"/>
+                </ui:VisualElement>
+            </ui:VisualElement>
+        </ui:Foldout>
+        <ui:Foldout name="AnywherePageComputeTitle" class="foldout" text="Register Compute">
+            <ui:VisualElement class="form-row">
+                <ui:Label name="AnywherePageComputeNameLabel" class="form-row__label" text="Compute Name"/>
+                <ui:TextField name="AnywherePageComputeNameInput" class="form-row__input text-input" value="ComputerName-ProfileName"/>
+            </ui:VisualElement>
+            <ui:VisualElement class="form-row">
+                <ui:Label name="AnywherePageComputeIPLabel" class="form-row__label" text="IP Address"/>
```

Comment:
Please follow same casing for `Ip` across occurrences.

Reply:
renamed